src/libraries/Microsoft.Extensions.Configuration.Ini/src/PACKAGE.md

```diff
@@ -1,49 +1,44 @@
 ## About
 
+<!-- A description of the package and where one can find more documentation -->
+
 INI configuration provider implementation for [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration/). This package enables you to read configuration parameters from [INI files](https://en.wikipedia.org/wiki/INI_file). You can use [IniConfigurationExtensions.AddIniFile](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.iniconfigurationextensions.addinifile) extension method on `IConfigurationBuilder` to add INI configuration provider to the configuration builder.
 
-For more information, see the documentation: [INI configuration provider](https://learn.microsoft.com/dotnet/core/extensions/configuration-providers#ini-configuration-provider).
-
-## Example
-The following example shows how to read the application configuration from INI file.
```

Comment:
Looks like this one might need to be undone - it's removing some content and template is not filled out completely.